src/logger.py

```diff
@@ -32,11 +32,14 @@ def __init__(self, bot, cfg):
         self.bot = bot
         self.cfg = cfg
         self.channel = None
+        self.__initialised = False
 
     @commands.Cog.listener()
     async def on_ready(self):
```

Comment:
what else triggers `on_ready()` ?

Reply:
It is triggered when discord.py connects or reconnects to the discord API. https://discordpy.readthedocs.io/en/latest/api.html#discord.on_ready

Reply:
so should there be a log for every reconnect? i feel like thats somewhat useful